### PR TITLE
Translation from English to Spanish downloads.md

### PR DESCRIPTION
**Pull request by ackseriam**
Hello, I´m Ericka Simancas. This is my first contribution to the community. I hope that you can accept me. To continue contributing to this excellent project

### CODE REVIEW

File: _i18n/es/downloads.md [hunk 1]
```diff
@@ -134,7 +133,7 @@ If you'd prefer to use a blockchain bootstrap, instead of syncing from scratch,
 {% endfor %}
 
 </div>
-<a href="#" class="arrow-up"><i></i></a>
+<a href="#" class="arrow-up"><i></i></a>yo
```
Comment (rehrar) [inline _i18n/es/downloads.md]: Not quite sure why you have an extra 'yo' here? :)

File: _i18n/es/downloads.md [hunk 2]
```diff
@@ -3,8 +3,7 @@
 
 <div class="container description" markdown="1">
 
-If you need help choosing the correct application, please click [here](https://www.reddit.com/r/Monero/comments/64b5lf/what_is_the_best_monero_wallet/) for a quick answer, then select the appropriate release for your operating system below.
-Note: the SHA256 hashes are listed by the downloads for convenience, but a GPG-signed list of the hashes is at [getmonero.org/downloads/hashes.txt](https://getmonero.org/downloads/hashes.txt) and should be treated as canonical, with the signature checked against the appropriate GPG key in the source code (in /utils/gpg_keys).
+Si necesita ayuda para elegir la aplicación correcta, haga clic aquí para obtener una respuesta rápida, luego seleccione la versión apropiada para su sistema operativo. Nota: para mayor comodidad los hashes SHA256 se enumeran por las descargas, pero una lista GPG firmada de los hash está en getmonero.org/downloads/hashes.txt y debe tratarse como canónico, con la firma marcada con la clave GPG apropiada en el código fuente (en /utils/gpg_keys).
```
Comment (rehrar) [inline _i18n/es/downloads.md]: There's been some discussion going on regarding the formal vs informal Spanish, and I think we've agreed to go with the informal. Also, I think this formatting is a little off compared to the English one, if you haven't looked at how to do Markdown, you can [learn here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).

**Comment by rehrar** [comment]
Hey @ackseriam, great to have you here! I just did a quick review of your work with a couple of suggested changes. If you have any questions, or need any help, please let me know. Also, feel free to drop by the #monero-translations IRC channel, or on [Monero's Mattermost](https://mattermost.getmonero.org) instance (which relays to the translation channel.

I'm thrilled to have your first contribution here, and I hope you find Monero to be a welcoming, inviting community to contribute to. Thanks so SO much for your work. 🥇

**Comment by ackseriam** [comment]
Hello @rehrar , thanks for the recommendations, I will be more careful in the future. I am very happy to contribute to this excellent community. I made the changes to the suggested recommendations 
48be71c

**Comment by ackseriam** [comment]
Change made, @erciccione  thanks for the suggestion, I will be reviewing the content

**Comment by erciccione** [comment]
Thanks! :)

**Comment by ackseriam** [comment]
Hello,  @erciccione  I would like to know when I can do the merging of the changes. Thanks.

**Comment by erciccione** [comment]
Hi! your commits will be merged soon, usually maximum two weeks :)

**Comment by ackseriam** [comment]
Ok, thanks @erciccione

**Comment by luigi1111** [comment]
@ackseriam Hello, these commits should be squashed into one, similarly to what is requested in your other PR. If you need assistance with that, someone in IRC or Mattermost as @rehrar mentions should be able to help.

Thanks for your contributions!

**Comment by erciccione** [comment]
@ackseriam come chat on `#monero-translations` if you need help squashing. We can do it together

**Comment by ackseriam** [comment]
Thank you very much @erciccione I just have one question. How can I access #monero-translation haha Sorry for the question.

**Comment by ackseriam** [comment]
Sorry for so much inconvenience

**Comment by erciccione** [comment]
no worries, not a problem at all. It's a chat on freenode, you can access with a client or from [your browser](https://imgur.com/a/TqmUD). Just join the channel `#monero-translations` :)
The chat is also relayed on riot/matrix and [mattermost](https://mattermost.getmonero.org/)

**Comment by erciccione** [comment]
@ackseriam I cna solve this for you. See [my comment in #609 ](https://github.com/monero-project/monero-site/pull/609#issuecomment-370481344)

**Comment by erciccione** [comment]
@ackseriam All three commits are now squashed in one
@luigi1111 this PR is good to go now

**Pull request merged**